curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-94-almost-equilateral-triangles.md

```diff
@@ -12,37 +12,73 @@ It is easily proved that no equilateral triangle exists with integral length sid
 
 We shall define an <dfn>almost equilateral triangle</dfn> to be a triangle for which two sides are equal and the third differs by no more than one unit.
 
-Find the sum of the perimeters of all <dfn>almost equilateral triangle</dfn> with integral side lengths and area and whose perimeters do not exceed one billion (1,000,000,000).
+Find the sum of the perimeters of all <dfn>almost equilateral triangles</dfn> with integral side lengths and area and whose perimeters do not exceed `limit`.
```

Comment:
A quick question before I approve - do we need to use the `dfn` tag more than once?

Reply:
Most likely no, I guess the tag in the middle (in line 13) should stay and remove it from elsewhere.